README.md

```diff
@@ -2,3 +2,5 @@
 TestPublicRepo
 
 This is a test repository.
+
+This line is modified by a user.
```

Comment:
Why don't you change the word "user" to your name?

Reply:
I've accepted your request and added my name.